### PR TITLE
Adds an optional selinux boolean needed for using the rsync distributor

### PR DESCRIPTION
**Pull request by dkliban**
closes #2199
https://pulp.plan.io/issues/2199

### CODE REVIEW

File: server/selinux/server/pulp-celery.te [hunk 1]
```diff
@@ -158,10 +161,23 @@ optional_policy(`
...
 
-rsync_exec(celery_t)
+gen_tunable(pulp_manage_rsync, false)
+
+optional_policy(`
+    tunable_policy(`pulp_manage_rsync', `
+        # Pulp executes rsync over ssh. Pulp needs to write to /var directories. Allow creation of
+        # a socket file. Allow managing a socket.
+        rsync_exec(celery_t)
+        ssh_exec(celery_t)
```
Comment (beav) [inline server/selinux/server/pulp-celery.te]: I'm not all that familiar with the rsync distributor, why is ssh needed?

Comment (dkliban) [inline server/selinux/server/pulp-celery.te]: Rsync distributors run rsync over ssh to transfer files to a remote server [0]. 

[0] https://github.com/pulp/pulp/blob/master/server/pulp/plugins/rsync/publish.py#L101-L108

**Comment by beav** [comment]
this looks good to me. I don't have an EL6 machine to test on, but as long as it compiles ok there, I have no additional comment.

**Comment by dkliban** [comment]
The policy compiles successfully on EL6.

**Comment by mhrivnak** [comment]
On behalf of and with gratitude to our esteemed alumnus @beav , I add this LGTM label.

**Pull request merged**